indra/llplugin/llpluginprocessparent.h

```diff
@@ -207,7 +207,7 @@ class LLPluginProcessParent : public LLPluginMessagePipeOwner
 	apr_pollfd_t mPollFD;
 	static apr_pollset_t *sPollSet;
 	static bool sPollsetNeedsRebuild;
-	static LLMutex *sInstancesMutex;
+	static LLCoros::Mutex *sInstancesMutex;
```

Comment:
To be honest I'm not sure why `LLPluginProcessParent` isn't derived from `LLInstanceTracker`.